packages/components/src/components/tabs/_tabs.scss

```diff
@@ -178,6 +207,13 @@
     }
   }
 
+  .#{$prefix}--tabs--fixed
+    .#{$prefix}--tabs__nav-item:hover:not(.#{$prefix}--tabs__nav-item--disabled) {
+    @include carbon--breakpoint(md) {
+      background-color: $hover-ui;
```

Comment:
I know this is the token the spec has, but for some reason the color it's receiving is `#e5e5e5` instead of `#cacaca`, so the hover color is lighter and it should be darker. Not sure why the color doesn't match the token though. Is this the right token? @aagonzales 
<img width="739" alt="Screen Shot 2019-10-23 at 11 31 41 AM" src="https://user-images.githubusercontent.com/32556167/67414482-ccb0b180-f588-11e9-9638-9b8820544fee.png">

Reply:
good catch, it will need to use a different token. Use`$hover-selected-ui` (even though its not selected it gets it the correct color) 🤷‍♀